assets/prefabs/spork.prefab

```diff
@@ -0,0 +1,16 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking:spork”,
```

Comment:
The second last character in this line (”) is a stray quote. ” != "
Please fix all such quote characters or the prefab won't load